src/wallet/coin_selection.rs

```diff
@@ -641,32 +646,25 @@ impl BranchAndBoundCoinSelection {
             .filter_map(|(optional, is_in_best)| if is_in_best { Some(optional) } else { None })
             .collect::<Vec<OutputGroup>>();
 
-        let selected_amount = best_selection_value.unwrap();
-
-        // remaining_amount can't be negative as that would mean the
-        // selection wasn't successful
-        // target_amount = amount_needed + (fee_amount - vin_fees)
-        let remaining_amount = (selected_amount - target_amount) as u64;
-
-        let excess = decide_change(remaining_amount, fee_rate, drain_script);
+        let fee_amount = selected_utxos.iter().map(|u| u.fee).sum::<u64>();
 
-        Ok(BranchAndBoundCoinSelection::calculate_cs_result(
-            selected_utxos,
-            required_utxos,
-            excess,
-        ))
+        Ok((selected_utxos, fee_amount))
     }
+}
+
+/// Single Random Draw coin selection
+#[derive(Debug)]
```

Comment:
Same as the other empty CS algs, this should derive `#[derive(Debug, Default, Clone, Copy)]`

Reply:
This doesn't apply here anymore.